README.md

```diff
@@ -1,6 +1,35 @@
+<!-- Badge for License -->
+<div align="right">
+
+  [![](https://img.shields.io/github/license/romainl/idiomatic-vimrc.svg?style=flat-square)](./LICENSE)
+
+</div>
+
+<!-- Logo -->
+<p align="center">
+  <a href="https://github.com/romainl/idiomatic-vimrc">
+    <img src="https://i.imgur.com/C6mxszA.png" alt="Idiomatic VIMRC" height="150px" />
+  </a>
+</p>
+
+<!-- Title and Description -->
+<div align="center">
+
 # IDIOMATIC VIMRC
 
-Guidelines for sculpting your very own `~/.vimrc`.
+🔨 _Guidelines for sculpting your very own `~/.vimrc`_
```

Comment:
Please revert this line to its original state:

    Guidelines for sculpting your very own `~/.vimrc`.

Reply:
reverted in the amending commit